### PR TITLE
[POC] LPS-122873 | LPS-125846

### PR DESCRIPTION
**Pull request by dgarciasarai**

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by darquesdev** [comment]
Hi @dgarciasarai, I can't see anything about the ordering, I think we need to reconsider the solution. What would you need to sort the y-axis (stage)?

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java [hunk 1]
```diff
@@ -261,6 +265,14 @@ private AssetVocabularyMetric _getAssetVocabularyMetric(
 			Collectors.toMap(Bucket::getKey, Bucket::getDocCount));
 	}
 
+	private String _getColor(String key) {
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java:268]: For the shake of the POC, could you extract the color on the front-side using the category kay?

Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/data/provider/ContentDashboardDataProvider.java:268]: OMG!!!! I can do it the same way in frontend side 🤯 Monday issues...I'm sorry.

**Comment by dgarciasarai** [comment]
> Hi @dgarciasarai, I can't see anything about the ordering, I think we need to reconsider the solution. What would you need to sort the y-axis (stage)?

I don't know what has to happen with the ordering :/  Let's talk and make the requisites crystal clear. Thanks!

**Pull request closed without merging**